hw/top_earlgrey/data/ip/chip_rv_plic_testplan.hjson

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_rv_plic
+  testpoints: [
+    // PLIC integration tests:
+    {
+      name: chip_sw_plic_all_irqs
+      desc: '''Verify all interrupts from all peripherals aggregated at the PLIC.
+
+            The automated SW test enables all interrupts at the PLIC to interrupt the core. It uses
+            the `intr_test` CSR in each peripheral to mock assert an interrupt, looping through all
+            available interrupts in that peripheral. The ISR verifies that the right interrupt
+            occurred. This is used as a catch-all interrupt test for all peripheral integration
+            testing within which functionally asserting an interrupt is hard to achieve or not of
+            high value.
+            '''
+      stage: V2
+      si_stage: SV2
```

Comment:
`chip_sw_plic_sw_irq` and  `chip_sw_plic_all_irqs` can be considered smoke tests. There are purely software tests and doesn't require any major change to work with SiVal.

Reply:
Well, I'm pleased to hear that there isn't much implementation work to do. But we still need to avoid giving the lab too much work running the tests :-) Maybe it makes sense to mark one of them as "earlier" and make that one SV2 and the other SV3?

Reply:
Updated si_stage label